dedupe/api.py

```diff
@@ -548,7 +548,8 @@ def _loadIndices(self, settings_file):
             for predicate in full_predicate:
                 if hasattr(predicate, "index") and predicate.index is None:
                     predicate.index = predicate.initIndex()
-                    predicate.index._doc_to_id = doc_to_ids[predicate]
+                    doc_to_id_max_id = numpy.amax([v for k, v in doc_to_ids[predicate].items()])
+                    predicate.index._doc_to_id = defaultdict(itertools.count(doc_to_id_max_id).next, doc_to_ids[predicate])
                     if hasattr(predicate, "canopy"):
```

Comment:
this needs to be 

```
defaultdict(itertools.count(doc_to_id_max_id+1).next, doc_to_ids[predicate])
```
to start counting **after** the current max id.

We also need to handle both python 2 and 3 like here: 
https://github.com/datamade/dedupe/blob/aa2d50af1a26be3f6ae03a262c1b750484a1f473/dedupe/tfidf.py#L14-L17